src/Internal.AspNetCore.BuildTools.Tasks/Internal.AspNetCore.BuildTools.Tasks.csproj

```diff
@@ -18,7 +18,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NETStandard.Library" Version="1.6.0" PrivateAssets="All" />
+    <PackageReference Include="NETStandard.Library" Version="$(NetStandardImplicitPackageVersion)" PrivateAssets="All" />
```

Comment:
Please revert the version bumps. These versions were picked specifically to align with what is in MSBuild for .NET Core 1.0. A dependency version bump in tools packages should result in a tools version bump. This may break patch builds. We'll take care of the upgrade in #200